__tests__/constants.js

```diff
@@ -10,3 +10,22 @@ test('getPathKey', () => {
   expect(getPathKey('linux', {})).toBe('PATH');
   expect(getPathKey('darwin', {})).toBe('PATH');
 });
+
+test('isProduction', () => {
```

Comment:
Protip: you can have `describe('isProduction')` at the top that wraps all this and then put each separate scenario in their own `test()` calls.

Btw. changing globals like `process.env.NODE_ENV` in tests is a bit dangerous since we run tests in parallel. It may yield to other problems. A no-to-great way is to define `isProduction` as follows that lets you do dependency injection:
```
function isProduction(env = process.env) {
    return env.NODE_ENV === 'production';
}
```